guideline-04.md

```diff
@@ -1,12 +1,13 @@
-<h4>4. Code must be (mostly) human readable.</h4>
+<h4>4. Original source code and  build tools must be publicly available.</h4>
 
-Obscuring code by hiding it with techniques or systems similar to <code>p,a,c,k,e,r</code>'s obfuscate feature, uglify's mangle, or unclear naming conventions such as <code>$z12sdf813d</code>, is not permitted in the directory. Making code non-human readable forces future developers to face an unnecessary hurdle, as well as being a common vector for hidden, malicious code.
-
-We require developers to provide public, maintained access to their source code and any build tools in one of the following ways:
+We require developers to provide public, maintained access to their original source code and any build tool configuration files and custom build tools in one of the following ways:
 
 <ul>
-    <li>Include the source code in the deployed plugin</li>
-    <li>A link in the readme to the development location</li>
+<li>Include the source code any build tool configuration files and custom build tools in the deployed plugin</li>
+<li>A link in the readme to the development location<li></li>
 </ul>
 
-We strongly recommend you document how any development tools are to be used.
+We strongly recommend you document how any build tools are to be used.
+
+Original source code must be human readable and not be deliberately obscured.
```

Comment:
Would add some additional cases as we saw some people being so prolific that they are able to read and edit minified code.

As for example:
Original source code must be human readable and not be deliberately obscured, minified, omitting comments, using unclear naming conventions and/or making use of other techniques that make it difficult to read, understand and/or edit.

Reply:
"omitting comments, using unclear naming conventions"

Isnt this a bit subjective? Especially comments. When is a comment in a the original source code omitted or just not written?

Reply:
I have mode some wording that I think covers those concerns

Reply:
Looks good, thanks :D